content/articles/compose-for-desktop/index.md

```diff
@@ -0,0 +1,347 @@
+                    singleLine = true, modifier = Modifier.fillMaxWidth(),
+                )
+
+                Spacer(modifier = Modifier.height(16.dp))
+                TextField(
+                    value = textFieldStatePassword,
+                    label = {
+                        Text("Enter your Password")
+                    },
+                    onValueChange = {
+                        textFieldStatePassword = it
+                    },
+                    keyboardOptions = KeyboardOptions(
+                        capitalization = KeyboardCapitalization.None,
+                        autoCorrect = false,
+                        keyboardType = KeyboardType.Password,
+                        imeAction = ImeAction.Next
+                    ),
+                    visualTransformation = PasswordVisualTransformation(),
+                    singleLine = true, modifier = Modifier.fillMaxWidth(),
+                )
+
+                Spacer(modifier = Modifier.height(16.dp))
+
+                Button(onClick = {
+                    scope.launch {
+                        scaffoldStateRegister.snackbarHostState.showSnackbar("Button Clicked $textFieldStatePassword")
+                    }
+                }){
+                    Text("Register")
+                }
+            }
+        }
+    } 
+
+}
+```
+Register class implementation is just the same as the `LoginScreen` but with more text fields hence we shall use the same explanation as the one above.
+
+![register-screen](/engineering-education/compose-for-desktop/register-screen.png)
+
+#### Step 5 - Main Class
+In this class, we will create the objects of our classes and use the object to call the composable methods.
+
+```kotlin
+@Composable
+@Preview
+fun App() {
+    MaterialTheme {
+        // LoginScreen object
+        val login = LoginScreen()
+        login.LoginScreen()
+
+        // RegisterScreen object
+        val register = RegisterScreen()
+        register.RegisterScreen()
+    }
+}
+
+fun main() = application {
+    Window(onCloseRequest = ::exitApplication) {
+        App()
+    }
+}
+```
+
+Since kotlin is considered a multiplatform language you can build a whole desktop application with Kotlin only. In this tutorial, we have learned how to build auth screens. 
+
+This project can be found in my [GitHub](https://github.com/Collince-Okeyo/UserAuthentication) repositories. You can fork and clone it in your machine and learn how to create the screens with Compose for Desktop implementation. 
+
+To learn more about building desktop apps with Compose for Desktop visit [here](https://github.com/JetBrains/compose-jb/tree/master/tutorials/Getting_Started).
+
+### Conclusion
+Reading this article will help in getting started with Compose for Desktop apps. To learn more about building desktop apps with this technology you will have to do a lot of practice. If you love creating web applications, with Compose for Web, you can create awesome websites as well.
```

Comment:
~~Reading this article will help in getting started with Compose for Desktop apps.~~ This sentence will fit more in an introduction than in a conclusion, replace it.

 with this technology, (add a comma).

Reply:
Add a further reading section to your article.